tsinfer/inference.py

```diff
@@ -1950,29 +1950,35 @@ def post_process(
     ts,
     *,
     split_mrca=None,
+    keep_flanks=None,
     # Parameters below deliberately undocumented
     warn_if_unexpected_format=None,
     simplify_only=None,
 ):
     """
-    post_process(ts, *, split_mrca=None)
+    post_process(ts, *, split_mrca=None, keep_flanks=None)
```

Comment:
It's a bit odd having positive and negative things done by default here (i.e, you do ``split_mrca`` by default but you don't ``keep_flanks``). It would be easier to remember if these were all positive statements which are things you do by default, so maybe rename to ``trim_flanks`` instead?

Reply:
Yes, this is a good point and one of the things I was wondering about. Do we want `split_mrca=None, trim_flanks=None` or the inverse (something like `keep_mrca=None, keep_flanks=None`). I guess the first is fine?

Reply:
Positive things that we do by default and we can optionally switch off is the simplest here I think.

Reply:
I think we shouldn't use the word "trim" though, as the "normal" tskit trimming functions `.ltrim` and `.rtrim` actually change the coordinate system. Here we are simply removing topology. So perhaps "delete_flanks" or "erase_flanks" or "omit_flanks"? Do you have any preference @jeromekelleher ?

Reply:
I've switched to `erase_flanks` as I think erasing gives the right mental picture.